feature/gribi/ate_tests/gribi_scaling/gribi_scaling_test.go

```diff
@@ -301,6 +301,10 @@ func createVrf(t *testing.T, dut *ondatra.DUTDevice, d *oc.Root, vrfs []string)
 			gnmi.Replace(t, dut, gnmi.OC().NetworkInstance(vrf).Config(), i)
 			nip := gnmi.OC().NetworkInstance(vrf)
 			fptest.LogQuery(t, "nonDefaultNI", nip.Config(), gnmi.GetConfig(t, dut, nip.Config()))
+		} else if vrf == *deviations.DefaultNetworkInstance {
+			i := d.GetOrCreateNetworkInstance(vrf)
+			i.Type = oc.NetworkInstanceTypes_NETWORK_INSTANCE_TYPE_DEFAULT_INSTANCE
+			gnmi.Replace(t, dut, gnmi.OC().NetworkInstance(vrf).Config(), i)
```

Comment:
Creation of the default VRF is not necessary and should be controlled by ExplicitInterfaceInDefaultVRF for other vendor. I can find similar configuration example from other test case. Please modify accordingly.